cwms-data-api/src/main/java/cwms/cda/api/timeseriesprofile/TimeSeriesProfileParserCatalogController.java

```diff
@@ -98,18 +98,14 @@ public void handle(@NotNull Context ctx) {
             List<TimeSeriesProfileParser> tspParsers = tspParserDao.catalogTimeSeriesProfileParsers(locationId,
                     officeIdMask, parameterIdMask, true);
             String acceptHeader = ctx.header(Header.ACCEPT);
-            // Uses manual JSON array formatting due to serialization issues with List for TimeSeriesProfileParser Type handling.
+            // Uses manual JSON array formatting due to serialization issues
```

Comment:
These comments are not relevant here. The link to the GitHub Issue should go with your custom parsing code overrides.

Reply:
Moved comments to serializer class and appropriate DTO serializer annotations